test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -98,7 +98,7 @@ def test_update_upload_status_yaml_file(setup_and_teardown):
     params = setup_and_teardown
     assembly_dir = params.assembly_dirs[0]
     assembly_name = ASSEMBLY_NAMES[0]
-    assembly_tuple = workspace_uploader._AssemblyTuple(
+    assembly_tuple = workspace_uploader._WSObjTuple(
```

Comment:
Might change assembly_* to WsObject_* as well while you are at it?

Reply:
I think for testing files, we are probably fine since as all those tests are specific to Assembly.

Reply:
Okay, we can change later when we test genomes.

Reply:
plus variable names should be snake_case in Python